src/components/dls/QuranWord/IndoPakWordText.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+
+type IndoPakWordTextProps = {
+  text: string;
+};
+
+const IndoPakWordText = (props: IndoPakWordTextProps) => {
+  const { text } = props;
```

Comment:
prefer to destruct in the function arg :)

Reply:
Is this a strong opinion >.<, this is the pattern that I'm used to but no strong opinions

Reply:
not a strong opinion but it's cleaner the other way 😓